cloud_init.tf

```
@@ -117,17 +117,16 @@ data "cloudinit_config" "cloud_init_tasks" {
   }
 
   part {
-    filename     = "link-nessus-agent.py"
+    filename     = "configure-falcon-sensor.py"
     content_type = "text/x-shellscript"
     content = templatefile(
-      "${path.module}/cloudinit/link-nessus-agent.py", {
-        nessus_agent_install_path = var.nessus_agent_install_path
-        nessus_groups             = join(",", var.nessus_groups)
-        nessus_hostname_key       = var.nessus_hostname_key
-        nessus_key_key            = var.nessus_key_key
-        nessus_port_key           = var.nessus_port_key
-        ssm_read_role_arn         = module.ssmreadrole.role.arn
-        # This is the region where the IPA instance is being created
+      "${path.module}/cloudinit/configure-falcon-sensor.py", {
+        falcon_sensor_install_path = var.crowdstrike_falcon_sensor_install_path
+        falcon_customer_id_key     = var.crowdstrike_falcon_sensor_customer_id_key
```

Correct alphabetical order:
```suggestion
        falcon_customer_id_key     = var.crowdstrike_falcon_sensor_customer_id_key
        falcon_sensor_install_path = var.crowdstrike_falcon_sensor_install_path
```